client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMetaFinalEmail.vue

```diff
@@ -0,0 +1,320 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
```

Comment:
When i send the final email, i get a notification that i should change the statement status to "Schlussmitteilung versendet". Is that possible? Or should we adjust the text?

Reply:
Mmh, no, good point, we can't change the status manually. I change the text.